Game/Game.cls

```diff
@@ -30,6 +30,15 @@ public with sharing class Game
     	return winner;
     }
 
+	public void fakeMethod(String blah){
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Avoid reassigning parameters such as 'blah' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://app.codescan.io/coding_rules#rule_key=sf%3AAvoidReassigningParameters)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Public method and constructor comments are Required [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://app.codescan.io/coding_rules#rule_key=sf%3ACommentRequired)